app/main.py

```diff
@@ -132,6 +132,20 @@ async def delete(request):
     status = await (dqueue.cancel(ids) if where == 'queue' else dqueue.clear(ids))
     return web.Response(text=serializer.encode(status))
 
+@routes.get(config.URL_PREFIX + 'history')
+async def list_queue(_):
+    history = [];
+    for _ ,v in dqueue.queue.saved_items():
+        obj = vars(v)
+        obj['list_type'] = 'queue'

```

Comment:
Wouldn't it be more useful to return the two lists separately, instead of one list with a type for each member? It'll make it more compact and sensible I think... I'm talking about this sort of response:
```
{
    "queue": [
        { ...item... },
        { ...item... },
    ],
    "done": [
        { ...item... },
        { ...item... },
    ],
}
```

Reply:
Indeed good idea, ills implement that and remove the `list_type` key